concepts/eventing/performance evaluation/nats-jetstream/investigation-discrepancy.md

```diff
@@ -0,0 +1,35 @@
+# Investigation of the discrepancy between sent and received events
+
+
+There were some test runs where the "Events Sent" didn't match "Events Received". This document investigates one such suspicious test run.
+
+> **Conclusion:** Once the minReplicas for the receiver (that is, the sink) was changed to six (so that it won't be scaling up during the tests), we haven't seen any job whose "Events Sent" didn't match "Events Received". Therefore, maybe the scaling up of the receiver caused the issue.
```

Comment:
```suggestion
### Conclusion

Once the minReplicas for the receiver (that is, the sink) was changed to six (so that it won't be scaling up during the tests), we haven't seen any job whose "Events Sent" didn't match "Events Received". Therefore, maybe the scaling up of the receiver caused the issue.
```